plugins/org.yakindu.sct.generator.c/src/org/yakindu/sct/generator/c/CCodeGeneratorStandardModule.java

```diff
@@ -62,8 +64,17 @@ protected void bindTracingProperty(GeneratorEntry entry, Binder binder) {
 		FeatureParameterValue traceExitFeature = entry.getFeatureParameterValue(FEATURE_TRACING,
 				PARAMETER_TRACING_EXIT_STATE);
 		boolean traceEnter = traceEnterFeature != null ? traceEnterFeature.getBooleanValue() : false;
-		boolean traceExit = traceExitFeature != null ? traceEnterFeature.getBooleanValue() : false;
+		boolean traceExit = traceExitFeature != null ? traceExitFeature.getBooleanValue() : false;//TODO: this line was previously:
+		//boolean traceExit = traceExitFeature != null ? traceEnterFeature.getBooleanValue() : false; // was this a bug?
```

Comment:
looks like a copy-paste bug for me, you can remove the comment

Reply:
Erledigt